### PR TITLE
Map Balloon

### PR DESCRIPTION
**Pull request by vNakamura**
affects: `@crave/farmblocks-map-balloon`

ISSUES CLOSED: #264

### CODE REVIEW

File: packages/map-balloon/AUTHORS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Crave Food Systems:
+    <https://github.com/CraveFood>
+
+Authors:
+    André L. C. Ferreira <alcf.2099@gmail.com> <alcferreira>
+    Fabricio C Zuardi <fabricio@fabricio.org> <fczuardi>
+    Felipe Gameiro <fgameiro>
+    Henrique Lobato Zago <lobato.zago@gmail.com> <henlz>
```
Comment (alcferreira) [inline packages/map-balloon/AUTHORS]: Remove Henrique

File: packages/map-balloon/src/ProductImages.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  background: linear-gradient(
+    180deg,
+    rgba(0, 0, 0, 0) 0%,
+    rgba(0, 0, 0, 0.48) 100%
+  );
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 24px;
+`;
```
Comment (alcferreira) [inline packages/map-balloon/src/ProductImages.js]: Move `Container` and `Gradient` to `styledComponents` folder

File: packages/map-balloon/src/styledComponents/FarmInfo.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  align-items: center;
+  justify-content: center;
+  height: 48px;
+  width: 48px;
+  min-width: 48px;
+  margin: 8px;
+  border: 1px solid rgba(0, 0, 0, 0.16);
+  border-radius: 4px;
+  background-color: #f6f8f9;
+`;
```
Comment (alcferreira) [inline packages/map-balloon/src/styledComponents/FarmInfo.js]: Move `Container` and `Icon` to `styledComponents` folder

File: packages/map-balloon/src/styledComponents/Balloon.js [hunk 4]
```diff
@@ -25,9 +25,10 @@ const open = keyframes`
   }
 `;
 
+const animation = css`animation: ${open} 0.3s ease-in-out;`;
+
 const Balloon = styled(Card)`
-  animation: ${open} 0.3s ease-in-out;
-  position: absolute;
+  ${props => props.animated && animation} position: absolute;
```
Comment (alcferreira) [inline packages/map-balloon/src/styledComponents/Balloon.js]: please add a `;` after animation `${props => props.animated && animation};`
so prettier brakes the line

File: packages/map-balloon/LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Crave Food Systems and AUTHORS
```
Comment (fczuardi) [inline packages/map-balloon/LICENSE]: 2018

File: packages/map-balloon/README.md [hunk 6]
```diff
@@ -62,12 +62,13 @@ farm symbol, and that the class name to include that icon is `.wg-purveyor`.
...
+  balloon will show a smaller version with no caption
+* `caption`: string (required if `logo` is not given). Text to show under the
+  images.
+* `imageSet`: array (required if `logo` is not given). Items should be objects
   with the following properties:
-  * `name`: string (required). Name of the product.
+  * `text`: string (required). Text to show over the the image.
   * `image`: string (required). Path to product image.
 * `open`: boolean. Defines if the balloon wil be visible. Otherwise only the pin
   will be rendered
```
Comment (fczuardi) [inline packages/map-balloon/README.md]: 👍

**Pull request merged**